src/ros_license_toolkit/checks.py

```diff
@@ -34,7 +35,7 @@ class Check:
     def __init__(self: 'Check'):
         """Initialize a check."""
         # overall success of this check
-        self.success: bool = False
+        self.success: Status = Status.FAILURE
```

Comment:
```suggestion
        self.status: Status = Status.FAILURE
```
Not sure if this is a good name. But I feel like success won't work any more